examples/redis/README.md

```diff
@@ -0,0 +1,15 @@
+# Splunk Redis+SignalFX Example
```

Comment:
nit, "SignalFx" is now part of Splunk Observability Cloud, and the redis receiver isn't branded as Splunk's. Maybe something like:

```suggestion
# Redis Infrastructure Monitoring Example
```